src/App.js

```diff
@@ -1,24 +1,39 @@
-import logo from './logo.svg';
 import './App.css';
+import CountriesList from './components/CountriesList';
+import CountryDetails from './components/CountryDetails';
+import Navbar from './components/Navbar';
+import { Routes, Route } from 'react-router-dom';
+import { useEffect, useState} from 'react';
+import countriesArr from "./countries.json"
+import axios from 'axios';
 
 function App() {
+
+ 
+  const [countries, setCountries] = useState(countriesArr);
```

Comment:
this is correct with json file, but when you want to use the data from API instead of the data from json, I would set an empty array for the initial state
`const [countries, setCountries] = useState([]);`